File: Linguagem_C/Algoritmos/LogicaParaDesenvolvimentoDeProgramacaoDeComputadores/capitulo_3/exercicio3C.c
#include <stdio.h>
#include <math.h>

double calculaVolume(double raio,double altura){
  return 3.14159*pow(raio,2)*altura;
}

int main(void){
  
  double raio,altura;

  printf("Digite o raio da lata de oleo:");
  scanf("%lf",&raio);

  printf("Digite a altura da lata de oleo:");
  scanf("%lf",&altura);
  
  printf("O volume da lata de oleo é: %.2fL\n",calculaVolume(raio,altura));
  return 0;
}

File: Python/Algoritmos/LogicaParaDesenvolvimentoDeComputadores/Capitulo_3/exercicio3C.py
import math
"""
 Calcular e apresentar o volume de uma lata de oleo.
 O usuario deve inserir a altura e o raio da lata
"""

altura = float(input("Digite a altura da lata de oleo:"))
raio = float(input("Digite o raio da lata de oleo:"))

volume = math.pi * math.pow(raio,2) * altura 

print(f"O volume da lata de oleo eh: {volume}");
